### DISCUSSION TITLE
License files for ./bootloader/waflib and ./bootloader/zlib

### DISCUSSION THREAD

**Discussion by kcehvoin1** [pyinstaller/pyinstaller#8558] [Features & Development]
Hi all, 

I'd like to start using PyInstaller for a project but would like to confirm the licenses for `waflib` and `zlib` before proceeding.

I took a look at [COPYING.txt](https://github.com/pyinstaller/pyinstaller/blob/develop/COPYING.txt) but I wasn't able to parse out the relevant Licenses for those exact directories. 

Would it be possible to explicitly add a License file under each of those directories?

Thanks in advance!
Kevin

**Comment by rokm** [comment]
> Would it be possible to explicitly add a License file under each of those directories?

It would be, and we should add them. 

These are the license files if you want to open a PR before we get around to it:

https://github.com/madler/zlib/blob/develop/LICENSE
https://gitlab.com/ita1024/waf/-/blob/master/LICENSE

**Comment by kcehvoin1** [comment]
Thanks for the prompt reply @rokm.

Here are the pull requests:
zlib - https://github.com/pyinstaller/pyinstaller/pull/8560
waflib - https://github.com/pyinstaller/pyinstaller/pull/8561
